### DISCUSSION TITLE
Where do we begin?

### DISCUSSION THREAD

**Discussion by theshmunguss1** [theshmunguss1/RamOS#3] [Announcements]
How should we start coding RamOS? What should we start coding? If anyone has any ideas please discuss them in the comment section.

**Comment by YesTheProxy** [comment]
Ok We need to start deploying some proxies (I'll do Doge v4, Rammerhead, and possibly Utopia)

**Reply by YesTheProxy** [reply]
https://ram-networks-doge.vercel.app/   Doge v4

**Reply by theshmunguss1** [reply]
I've been trying to do Astroid V3

**Reply by YesTheProxy** [reply]
I'll Try it I guess

**Reply by YesTheProxy** [reply]
So I deployed https://rammerstroidv3.vercel.app/ and it looked to work, BUT when you try to use something it won't work so, I don't thing we should use Astroid v3

**Reply by theshmunguss1** [reply]
>when you try to use something it won't work

By that I assume you mean as an embed on the site? Astroid does not want to be embedded. @DJshelfmushroom you know more than us, pretty please help

**Reply by YesTheProxy** [reply]
No, I mean that you can type an address and press enter (Or search) and It won't do anything
Oh and https://mt64f6-8080.csb.app/ here's an incognito deployment

**Reply by DJshelfmushroom** [reply]
lmao asteroid is blocked for me I can't help, unless you can give me more info

**Reply by DJshelfmushroom** [reply]
nvm? its unblocked? idk i can help if you tell me what you want

**Reply by YesTheProxy** [reply]
can you deploy It and make it work please?

**Reply by theshmunguss1** [reply]
try https://happyholidays.shekinahphotography.com/

**Reply by theshmunguss1** [reply]
Nevermind actually. Got it up and running!

**Reply by YesTheProxy** [reply]
It worked! yayyyy

**Reply by YesTheProxy** [reply]
I think we should deploy the site on vercal.

**Comment by YesTheProxy** [comment]
Hey @DJshelfmushroom is there any way to open the "apps" in pop-ups in the site?
